docs/Versioning.md

```diff
@@ -2,8 +2,7 @@
 
 ## API
 ONNX Runtime follows [Semantic Versioning 2.0](https://semver.org/) for its public API.
-Each release has the form MAJOR.MINOR.PATCH. The meanings of MAJOR, MINOR and PATCH are
-same as what is described in the semantic versioning doc linked above.
+Each release has the form MAJOR.MINOR.PATCH, adhering to the definitions from the linked semantic versioning doc.
```

Comment:
But the C/C++ API has a different version system. It only has a major version, no minor or patch part. So I need we need to re-org this paragraph.  None of our API follows Semantic Versioning.